src/jdk.httpserver/share/classes/com/sun/net/httpserver/spi/HttpServerProvider.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2005, 2023, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Seems you changed the header of the wrong file

Reply:
Ah, thanks